lidar_litev3hp_sensor1/lidar_litev3hp_sensor1.ino

```diff
@@ -0,0 +1,77 @@
+/*------------------------------------------------------------------------------
+
+  Driver code is based on Garmin LIDAR-Lite v3hp Arduino Library (https://github.com/garmin/LIDARLite_Arduino_Library) 
```

Comment:
Please add a section to the README.md file in this repo that mentions that this library is used and mentions that the library is licensed under the Apache 2.0 License.